packages/jest-snapshot/src/SnapshotResolver.ts

```diff
@@ -78,11 +80,12 @@ function createDefaultSnapshotResolver(): SnapshotResolver {
 
 async function createCustomSnapshotResolver(
   snapshotResolverPath: Config.Path,
-  localRequire: LocalRequire,
+  localRequire: LocalRequire<SnapshotResolver>,
 ): Promise<SnapshotResolver> {
-  const custom: SnapshotResolver = interopRequireDefault(
-    await localRequire(snapshotResolverPath),
-  ).default;
+  const custom: SnapshotResolver = await localRequire(
+    snapshotResolverPath,
+    true,
```

Comment:
Add a boolean parameter to localRequire to support `interopRequireDefault` here.